### PR TITLE
[OKD-Core] Introduce baremetal host list filters

### PR DESCRIPTION
**Pull request by jtomasek**
Depends on https://github.com/kubevirt/web-ui-components/pull/257

### CODE REVIEW

File: frontend/public/metalkube/components/host/menu-actions.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Kebab } from '../utils/okdutils';
+
+const menuActionDrainHost = (kind, host) => ({
+  hidden: false, // TODO(jtomasek): use canDrainHost selector
+  label: 'Drain Host',
+  callback: () => {
```
Comment (suomiy) [inline frontend/public/metalkube/components/host/menu-actions.ts]: @mareklibra @rawagner  is it okay to have actions in the UI which don't do anything?

Comment (rawagner) [reply frontend/public/metalkube/components/host/menu-actions.ts]: @suomiy Im not in favour, but If they need it for the demo..

Comment (mareklibra) [reply frontend/public/metalkube/components/host/menu-actions.ts]: Considering context of the effort, I would not block on this.

Comment (suomiy) [reply frontend/public/metalkube/components/host/menu-actions.ts]: ok

**Comment by mareklibra** [comment]
This is still WIP but in a compact shape. Considering context, this can be merged and updated by follow-ups.

**Pull request merged**